controllers/etcd_controller.go

```diff
@@ -740,150 +746,79 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 }
 
 func (r *EtcdReconciler) addFinalizersToDependantSecrets(etcd *druidv1alpha1.Etcd) error {
-	if etcd.Spec.Backup.Store != nil {
-		storeSecret := corev1.Secret{}
-		// If secretRef is not nil, then we error out if its not found as well.
-		if etcd.Spec.Backup.Store.SecretRef != nil {
-			if err := r.Client.Get(context.TODO(), types.NamespacedName{
-				Name:      etcd.Spec.Backup.Store.SecretRef.Name,
-				Namespace: etcd.Namespace,
-			}, &storeSecret); err != nil {
-				return err
-			}
-			if finalizers := sets.NewString(storeSecret.Finalizers...); !finalizers.Has(FinalizerName) {
-				logger.Infof("Adding finalizer (%s) for secret %s by etcd (%s)", FinalizerName, storeSecret.GetName(), etcd.Name)
-				storeSecretCopy := storeSecret.DeepCopy()
-				finalizers.Insert(FinalizerName)
-				storeSecretCopy.Finalizers = finalizers.UnsortedList()
-				if err := r.Update(context.TODO(), storeSecretCopy); err != nil {
-					return err
-				}
-			}
-		}
-	}
 
-	// As the secrets inside TLS field are required, we error in case it is not found as well.
+	tlsSecrets := []string{}
```

Comment:
```suggestion
	secrets := []string{}
```
The backup secret is not TLS secret 😉